scanpipe/pipes/d2d.py

```diff
@@ -442,8 +438,24 @@ def create_package_from_purldb_data(project, resources, package_data):
             # and its descendants.
             path = f"{resource.path}/"
             lookups |= Q(path__startswith=path)
+    if lookups != Q():
```

Comment:
```suggestion
    if lookups:
```

This should be enough since 
```
>>> bool(Q())
False
```